commands/core/upgrade.go

```diff
@@ -71,7 +77,7 @@ func upgradePlatform(pm *packagemanager.PackageManager, platformRef *packagemana
 	}
 	latest := platform.GetLatestRelease()
 	if !latest.Version.GreaterThan(installed.Version) {
-		return fmt.Errorf("platform %s is already at the latest version", platformRef)
+		return ErrAlreadyLatest
```

Comment:
this way we avoid returning an error if users ask to upgrade a core that's already latest